src/AppInstallerCLICore/Argument.cpp

```diff
@@ -18,7 +18,7 @@ namespace AppInstaller::CLI
         case Args::Type::Query:
             return Argument{ "query", 'q', Args::Type::Query, Resource::String::QueryArgumentDescription, ArgumentType::Positional};
         case Args::Type::Manifest:
-            return Argument{ "manifest", 'm', Args::Type::Manifest, Resource::String::ManifestArgumentDescription, ArgumentType::Standard, Argument::Visibility::Help };
+            return Argument{ "manifest", 'm', Args::Type::Manifest, Resource::String::ManifestArgumentDescription, ArgumentType::Standard, Argument::Visibility::Help, Settings::TogglePolicy::LocalManifestFiles };

```

Comment:
>Settings::TogglePolicy::LocalManifestFiles [](start = 167, length = 42)

There you go making a case for these to be back in this shared location again.

If in the process of this work you put this on an argument that is constructed in a command, please move all instances here.  Any arguments with a feature flag should also be in here.

Basically, if the price of missing on is a policy is not properly enforced, then it should go through this method.  Please enforece that by making the constructor that takes the feature/policy private to force this path.  Hopefully that will make someone who is copying an existing argument do the right thing.

Reply:
I made the constructors private. I didn't find anything using policies or features that wasn't already there.